src/api/app/controllers/webui/image_templates/interconnects_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Webui
+  module ImageTemplates
+    class InterconnectsController < Webui::WebuiController
+      def index
+        @projects = Project.image_templates
+        respond_to do |format|
+          format.xml
```

Comment:
@bgeuken i agree that its better to not split the controller logic between API and Webui, but has this already been discussed and is everybody aware that this is the direction we are heading?